### ISSUE TITLE
Metrics Explained?

### ISSUE THREAD

**Issue by hdparmar** [justinpinkney/stable-diffusion#88]
I'm kinda lost on the metrics, I tried to search for it in files where it probably writes into csv with imwrite but couldn't find it.
It would mean a lot if someone explained me this!
Do you know what all this stands for?
I can guess the common ones like train/loss_epoch and stuff but what is?

train/loss_simple_epoch
train/loss_vlb_epoch
val/loss_simple
val/loss_vlb
val/loss_vlb_ema
val/loss_simple_ema
val/loss_ema?

**Comment by hdparmar** [comment]
Okay, I think this is it, vlb might be variational lower bound. And I assume, the loss_simple is based on MSE loss that the model is using

**Comment by hdparmar** [comment]
Gotcha, vlb is the variational lower bound of VAE and ema is well exponential moving average loss.
